samples/nrf9160/lwm2m_client/README.rst

```diff
@@ -65,7 +66,7 @@ on the nRF9160-DK board using the default configuration :file:`prj.conf`.
 You will need to tell the sample what LwM2M server to use by editing the
 following line in the configuration you've chosen::
 
-    CONFIG_APP_LWM2M_SERVER="{IP address of LwM2M server}"
+    CONFIG_APP_LWM2M_SERVER="{leshan.eclipseprojects.io}"
```

Comment:
Are the { } required? I would guess that they indicate that this part must be replaced?
So that you would set:
CONFIG_APP_LWM2M_SERVER="leshan.eclipseprojects.io"

@eibe85 - You have tested it, can you clarify?

Reply:
This should be without { }. I checked with Eivind yesterday, but implemented the wrong version.

Reply:
Clarified this with Eivind yesterday, but implemented the old version. Fixed now.

Reply:
I used SES for testing and there you do not need to use "" for strings, I guess they are added for me. If a customer uses prj.conf files for configuring they will have to have this string in the file CONFIG_APP_LWM2M_SERVER="leshan.eclipseprojects.io" with the "" no {} inside the "".